client/me/purchases/manage-purchase/style.scss

```diff
@@ -301,16 +301,23 @@
 	}
 }
 
-.manage-purchase__renew-button {
-	@include breakpoint-deprecated( '<660px' ) {
-		margin-top: 16px;
-		width: 100%;
-	}
+.manage-purchase__renew-upgrade-buttons {
+	display: none;
```

Comment:
This will also hide the renew button for mobile devices, no? Is that intentional?

Reply:
Yes -- and the main reason for this is two-fold 🙂 

1. We have action buttons located below the product card as well; and
2. currently on mobile, the buttons in the upper-right wrap to the bottom of the card, just above the _other_ buttons.

So, to avoid redundancy, I thought it made the most sense to completely hide these buttons unless the screen is wide enough to support them being in both places.